LICENCE.txt

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017 Shubham Dighe
+Copyright (c) 2017 Yagnesh L Pazhaniyappan
```

Comment:
Lol nice

Reply:
I am a noobie with VCS. I did a mistake. Didn't know what's the norm. 
Its a wrong practice which i realised later. 
I own up to the accusation that this is blatant plagiarism and when i learnt that, i back pedalled.